app/core/utils.coffee

```diff
@@ -1,5 +1,77 @@
 slugify = _.str?.slugify ? _.string?.slugify # TODO: why _.string on client and _.str on server?
 
+translatejs2cpp = (jsCode, fullCode=true) ->
```

Comment:
This function is missing tests in this PR.

Reply:
oh, my fault! i add /app directory into git repo and missing /test directory :joy:, updated